### PR TITLE
Correct typos, fix grammar, wrap at 80 columns, general cleanup.

### PR DESCRIPTION
**Pull request by atoponce**
Noticed some typos and odd paragraph behavior in the README.md. This cleans up plenty of typos, wraps all lines at 80 columns, where appropriate, removes unnecessary newlines, addresses English grammar, expands "JS" to JavaScript, and fixes one invalid Markdown syntax.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,17 @@
-
 [![Build Status](https://travis-ci.org/drand/drand.svg?branch=master)](https://travis-ci.org/drand/drand)
 
 # Drand - A Distributed Randomness Beacon Daemon
-
-Drand (pronounced "dee-rand") is a distributed randomness beacon daemon written
```
Comment (nikkolasg) [inline README.md]: Sounds a bit harsh to remove it alltogether no ? I'd like to keep a very short intro of what drand actually is.

File: README.md [hunk 2]
```diff
@@ -571,74 +560,86 @@ contacted drand node. If the encryption is not correct, the command outputs an
...
 
 ## DrandJS
-
-To facilitate the use of drand's randomness in JavaScript-based applications, we
-provide [DrandJS](https://github.com/PizzaWhisperer/drandjs). The main method
-`fetchAndVerify` of this JavaScript library fetches from a drand node the latest
-random beacon generated and then verifies it against the distributed key.  For
-more details on the procedure and instructions on how to use it, refer to the
+To facilitate the use of drand's randomness in JavaScript-based applications,
+we provide [DrandJS](https://github.com/PizzaWhisperer/drandjs). The main
```
Comment (nikkolasg) [inline README.md]: New url is https://github.com/drand/drandjs

File: README.md [hunk 3]
```diff
@@ -158,23 +145,21 @@ The script spins up 5 local drand nodes using Docker and produces fresh
 randomness every 10 seconds.
```
Comment (nikkolasg) [inline README.md]: This is just making me realize we should add a link to the readme of the demo in the folder `demo/` (it explains how to query the nodes as well)

File: README.md (hunk 1 above)
Comment (atoponce) [reply README.md]: Oops. Honest accident.

File: README.md (hunk 2 above)
Comment (atoponce) [reply README.md]: Updated.

File: README.md (hunk 3 above)
Comment (atoponce) [reply README.md]: Added.

**Comment by nikkolasg** [comment]
Awesome thanks a lot @atoponce  !!

**Pull request merged**